### PR TITLE
Add operator config for GCP Pubsub Publish operator

### PR DESCRIPTION
**Pull request by brighton1101**
This PR makes the GCP Pubsub Publish operator available in boundary-layer.

https://airflow.readthedocs.io/en/latest/_api/airflow/providers/google/cloud/operators/pubsub/index.html

### CODE REVIEW

**Comment by dossett** [comment]
We discussed (offline) changing this PR to match Airflow 1.10.3.

File: boundary_layer_default_plugin/preprocessors.py [hunk 1]
```diff
@@ -216,3 +218,57 @@ def _template_context(raw_args):
...
+                'Error in preprocessor {} for argument `{}`: {}'.format(
+                    self.type,
+                    arg,
+                    str(e)))
+        return bin_string
+
+    def _json_handler(self, arg):
+        return json.dumps(arg)
+
+    def _is_json(self, arg):
```
Comment (dossett) [inline boundary_layer_default_plugin/preprocessors.py:273]: Would `_is_dict` be clearer?  It's not literally a json string or object if this returns true.

Comment (brighton1101) [reply boundary_layer_default_plugin/preprocessors.py:273]: 👍 makes sense to me, will make that change

**Comment by brighton1101** [comment]
@dossett That change is now reflected, thank you for the review! Not sure how boundary-layer releases work, but lmk if I can be of any assistance in any way.

**Comment by dossett** [comment]
Thanks @brighton1101 I'll merge now and release it tomorrow.

**Pull request merged**